src/yki/handler/user.clj

```diff
@@ -0,0 +1,20 @@
+(ns yki.handler.user
+  (:require
+    [compojure.api.sweet :refer [api context GET]]
+    [integrant.core :as ig]
+    [ring.util.http-response :refer [ok bad-request internal-server-error]]
+    [yki.handler.routing :as routing]
+    [yki.registration.registration :as registration]
+    [yki.spec :as ys]
+    [yki.util.audit-log :as audit]))
+
+(defmethod ig/init-key :yki.handler/user [_ {:keys [db auth access-log user-config]}]
+  {:pre [(some? db) (some? auth) (some? access-log)]}
+  (api
+    (context routing/user-api-root []
+      :coercion :spec
+      :middleware [auth access-log]
+      (GET "/identity" {session :session}
+        (ok (or user-config (update-in session [:identity] dissoc :ticket))))
```

Comment:
Pienenä muutoksena vanhaan toteutukseen, tässä on mahdollista ottaa käyttäjä conffista, minkä havaitsin näppäräksi testailuissa.

Reply:
(huomiona myös, että tässä ei ole vielä poistettu samaa endpointtia auth-handlerista - tämä on lähinnä yhteensopivuutta vanhan version kanssa varten, poistan sen vielä erikseen)

Reply:
Tän `user-config`in sijasta mun mielestä tuon identiteetin feikkaamisen voisi mieluummin hoitaa kokonaan autentikaatio-middlewaren puolella. Esimerkiksi tuota `yki.middleware.no-auth/with-fake-oid`-middlewarea voisi pienten muokkausten jälkeen käyttää tässä. Vähentäisi esim. tuotanto-onnettomuuksien riskiä. Samaten siellä `yki.handler.registration` puolella voisi mun mielestä pyrkiä eroon tästä `user-config`-parametrista.